### PR TITLE
feat(Avatar): Added prop that makes the avatar become bigger while hovering it

### PR DESCRIPTION
**Pull request by orangeflame**
Add animation for student profile, when user hover on student avatar, avatar itself will scale bigger.

### CODE REVIEW

File: src/components/Avatar/index.jsx [hunk 1]
```diff
@@ -54,11 +54,15 @@ class Avatar extends PureComponent {
...
-    return <div style={this._style} className={classnames(styles.Avatar, this.props.className)} />;
+    const className = classnames(styles.Avatar, {
+      [styles.__animation]: this.props.animitaion,
+    }, this.props.className);
+    return <div style={this._style} className={className} />;
   }
 }
 
 Avatar.propTypes = {
+  animitaion: PropTypes.bool,
```
Comment (rubenillodo) [inline src/components/Avatar/index.jsx]: `animitaion` is misspelled. Let's change it to `animated`.

File: src/components/Avatar/style.postcss [hunk 2]
```diff
@@ -6,4 +6,12 @@
   border-radius: 50%;
   display: block;
   flex-shrink: 0;
+
+  &.__animation {
+    transition: all 0.3s ease-in-out;
```
Comment (rubenillodo) [inline src/components/Avatar/style.postcss]: You can use the variable `--transitionDurationBase` here.

File: src/components/Avatar/index.jsx [hunk 3]
```diff
@@ -54,11 +54,15 @@ class Avatar extends PureComponent {
   }
 
   render() {
-    return <div style={this._style} className={classnames(styles.Avatar, this.props.className)} />;
+    const className = classnames(styles.Avatar, {
+      [styles.__animation]: this.props.animitaion,
```
Comment (rubenillodo) [inline src/components/Avatar/index.jsx]: `this.props.animitaion` should change to `this.props.animated` here too.

File: src/components/StudentPicture/index.jsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ export const StudentPicture = ({ src, className, gender }) => <Avatar src={src}
     default={defaultAvatarImg}
     defaultMale={defaultAvatarMaleImg}
     defaultFemale={defaultAvatarFemaleImg}
+    animitaion
```
Comment (rubenillodo) [inline src/components/StudentPicture/index.jsx]: `animitaion` should be changed to `animated` here as well.

**Comment by orangeflame** [comment]
Remove the branch after pull request finishes

**Pull request merged**